kvbc/include/categorization/kv_blockchain.h

```diff
@@ -0,0 +1,60 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "updates.h"
+
+#include "kv_types.hpp"
+
+namespace concord::kvbc::categorization {
+
+class Block;
+class KeyValueBlockchain {
+ public:
+  // 1) Defines a new block
+  // 2) calls per cateogry with its updates
+  // 3) inserts the updates KV to the DB updates set per column family
+  // 4) add the category block data into the new block
+  BlockId addBlock(const Updates& updates) {
+    // Use new client batch and column families
+    // auto writeBatch = nativeClient->getBatch();
+    // const auto addedBlockId = getLastReachableBlockId() + 1;
+    Block newBlock;
+    // auto parentBlockDigestFuture = computeParentBlockDigest(blockId);
+
+    // Per category updates
+    for (const auto& [category_id, update] : updates.categoryUpdates) {
+      // https://stackoverflow.com/questions/46114214/lambda-implicit-capture-fails-with-variable-declared-from-structured-binding
+      std::visit(
+          [newBlock, category_id = category_id](auto& update) {
+            auto [catDbUpdates, blockUpdates] = handleCategoryUpdates(update, category_id);
```

Comment:
As discussed offline, we can pass the RocksDB write batch to `handleCategoryUpdates()` so that every category can use multiple column families.